src/vm_cpu.rs

```diff
@@ -296,6 +335,26 @@ pub struct ActiveVmCpu<'vcpu, 'pages, 'prev, T: GuestStagePagingMode> {
     parent_vcpu: Option<&'prev mut dyn VmCpuSaveState>,
 }
 
+#[cfg(target_feature = "v")]
+fn run_guest_vector(_state: *mut VmCpuState) {
```

Comment:
nit: No need for the `_` prefix since we're actually using the argument.

Reply:
will fix